lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -188,11 +188,36 @@ defmodule ExUnit.Runner do
       {:error, %{test_case | state: failed}}
   end
 
-  defp run_test(config, test, context) do
+  defp run_test(true, config, test, context) do
+    run_test(:stdio, config, test, context)
+  end
+
+  defp run_test(false, config, test, context) do
+    spawn_test(config, test, context)
+  end
+
+  defp run_test(device, config, test, context) do
+    ref = make_ref()
+    try do
+      ExUnit.CaptureLog.capture_log(device, fn ->
+        send self(), {ref, spawn_test(config, test, context)}
+      end)
+    catch
+      :exit, :noproc -> %{test | state: {:invalid, :no_logger}}
```

Comment:
We can go with invalid but we need to make the error message as clear as possible. Can the second element of the tuple be a string? I don't remember anymore. But we need to say "could not run test because it uses `@tag :capture_log` but the Logger application is not running".